app/javascript/src/components/Dashboard/Notes/NotesCard/Cards.jsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+
+import { MenuVertical, Clock } from "neetoicons";
+import { Tag, Tooltip, Avatar, Dropdown } from "neetoui";
+import { Header } from "neetoui/layouts";
+
+import { USER_AVATAR_URL } from "../constants";
+import { requiredDateFormat, relativeCreationTime } from "../utils";
+
+const CardDiv = ({ item }) => (
```

Comment:
The component name and the filename doesn't match. That's not right. Please read: https://academy.bigbinary.com/learn-react/pages/naming-conventions-in-react